arm/apimanagement/apioperations.go

```diff
@@ -47,20 +45,20 @@ func NewAPIOperationsClientWithBaseURI(baseURI string, subscriptionID string) AP
 // CreateOrUpdate creates a new API operation or updates an existing one.
 //
 // resourceGroupName is the name of the resource group. serviceName is the name
-// of the API Management service. apiID is aPI identifier. Must be unique in
+// of the API Management service. aPIID is aPI identifier. Must be unique in
```

Comment:
The reason this appeared is because of this... https://github.com/Azure/autorest/pull/2220 (just for reference)
In this PR, travis failed in the first commit with this message...
`arm/hdinsight/models.go:534:2: struct field ClusterUsersGroupDNs should be ClusterUsersGroupDNS`